roles/apache/tasks/main.yml

```diff
@@ -106,6 +107,16 @@
       - userdir.conf
       - welcome.conf
     notify: restart apache
+
+  - name: enable the mod_proxy_uwsgi module
+    template:
+      src: etc/httpd/conf.modules.d/43-mod_proxy_uwsgi.conf
+      dest: /etc/httpd/conf.modules.d/43-mod_proxy_uwsgi.conf
+      mode: 0644
+      owner: root
+      group: root
+    notify: restart apache
+
```

Comment:
This is done with adding `- proxy_uwsgi` to the list of modules to be enabled by a2enmod on ubuntu. This is on line 37.